code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -25,12 +25,16 @@
 	inherent_verbs = list(
 		/mob/living/carbon/human/proc/reconstitute_form,
 		/mob/living/carbon/human/proc/sonar_ping,
+<<<<<<< HEAD
 		/mob/living/carbon/human/proc/tie_hair,
 		/mob/living/proc/flying_toggle,
 		/mob/living/proc/flying_vore_toggle,
 		/mob/living/proc/start_wings_hovering,
```

Comment:
These three lines were removed, for reference. The reason it's got a head here is since lick_wounds is a Chompstation change,